lib/log.js

```diff
@@ -22,77 +28,89 @@ const levelColors = {
   60: 'bgRed'
 };
 
-function ConsoleStream(env) {
-  Writable.call(this, {objectMode: true});
+const console = new Console({
+  stdout: process.stdout,
+  stderr: process.stderr,
+  colorMode: false
+});
 
-  this.debug = env.debug;
-}
+class Logger {
+  constructor(options = {}) {
+    const silent = options.silent || false;
+    this._debug = options.debug || false;
 
-require('util').inherits(ConsoleStream, Writable);
+    this.level = INFO;
 
-ConsoleStream.prototype._write = function(data, enc, callback) {
-  const level = data.level;
-  let msg = '';
+    if (silent) {
+      this.level = WARN;
```

Comment:
Was this done intentionally? 
Even if the `silent` option is true, `hexo-log` display a log message higher than the `WARN` level log.

Reply:
Oh I forgot the behavior of `hexo-log`. `hexo-log` won't display any message if `silent` is true.

Reply:
Update: now the logger won't output anything if `silent` is set to `true`.